docs/intro/arch_overview/outlier.rst

```diff
@@ -69,6 +70,9 @@ being ejected and for what reasons. The log uses a JSON format with one object p
 time
   The time that the event took place.
 
+secs_since_last_action
+  The time that the last event took place. This time will be -1 for the first ejection.
```

Comment:
This text needs to work. It's the time in seconds since the last action took place (not the time), and it covers unejection as well as ejection.